misc/Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN echo "Go image: $GO_IMAGE"
 # Misc dependencies
 ENV HOME /opt    
 ENV DEBIAN_FRONTEND noninteractive    
-RUN apt-get update && apt-get install -y apt-utils curl git git-core bsdmainutils python3 python3-pip make bash libtool
+RUN apt-get update && apt-get install -y apt-utils curl git git-core bsdmainutils python3 python3-pip make bash libtool libboost-all-dev
```

Comment:
there was a recent change in go-algorand to use a sub-package of libboost to make installs faster on build/test machines

Reply:
while this is true, this PR https://github.com/algorand/go-algorand/pull/2717 is still failing on travis. I would wait for the author to finish fixing the PR before "copying" the solution.